### PR TITLE
New version: G3G4X5X6.ultimate-cube version 6.6.8

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65615)

### CODE REVIEW

File: manifests/g/G3G4X5X6/ultimate-cube/6.6.8/G3G4X5X6.ultimate-cube.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: G3G4X5X6.ultimate-cube
+PackageVersion: 6.6.8
```
Comment (Trenly) [inline manifests/g/G3G4X5X6/ultimate-cube/6.6.8/G3G4X5X6.ultimate-cube.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 6.6.6
```

File: manifests/g/G3G4X5X6/ultimate-cube/6.6.8/G3G4X5X6.ultimate-cube.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: G3G4X5X6.ultimate-cube
+PackageVersion: 6.6.8
```
Comment (Trenly) [inline manifests/g/G3G4X5X6/ultimate-cube/6.6.8/G3G4X5X6.ultimate-cube.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 6.6.6
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Seems they write the wrong version to ARP

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/65615#pullrequestreview-1034248545

**Review by Trenly — APPROVED** [review]

**Pull request merged**